src/locale/locales/fr/messages.po

```diff
@@ -1275,7 +1275,7 @@ msgstr "Modifier"
 
 #: src/screens/Settings/AppIconSettings.tsx:81
 msgid "Change app icon to \"{0}\""
-msgstr ""
+msgstr "Changer l’icône de l’application pour \"{0}\""
```

Comment:
Might it not make sense to use _guillemets_ here, as it's referring to the name of the app icon?

Reply:
Oh, good catch!